content/developer/api/version-8/JSON-API.adoc

```diff
@@ -321,14 +321,12 @@ DELETE {{your-SuiteCRM-url}}/V8/module/{moduleName}/{id}
 === Get relationship
 
 [source,php]
-GET {{your-SuiteCRM-url}}/V8/module/{moduleName}/relationships/{relation}
-
-Available parameters: fields, page, sort, filter
+GET {{your-SuiteCRM-url}}/V8/module/{moduleName}/{id}/relationships/{relation}
 
 Example:
 
 [source,php]
-V8/module/Accounts?fields[Accounts]=name,account_type&page[size]=4&page[number]=4
+V8/module/Accounts/11a71596-83e7-624d-c792-5ab9006dd493/relationships/aos_contracts
```

Comment:
More personal opinion but I would refrain from using actual ID values in documentation etc. as this may mislead some users (Has happened at least twice recently in my exp).